app/logic/Mail/SQL/SQLEMail.ts

```diff
@@ -363,6 +365,8 @@ export class SQLEMail {
     email.isSpam = sanitize.boolean(row.isSpam, false);
     email.threadID = sanitize.string(row.threadID ?? row.parentMsgID, null);
     email.downloadComplete = sanitize.boolean(row.downloadComplete, false);
+    let json = sanitize.json(row.json, {});
+    email.scheduling = sanitize.integer(json.scheduling, 0);
```

Comment:
This is incorrect, it should say
```suggestion
    email.scheduling = sanitize.integer(json.invitationMessage, 0);
```
So far I have not found any other problems.

Reply:
Thanks. Fixed in git commit 825bb5ce

Reply:
Yes I had suspected you would do that :wink: